pennylane/ops/op_math/controlled_ops.py

```diff
@@ -150,35 +168,58 @@ def _primitive_bind_call(
     def __init__(
         self,
         base,
-        control_wires: WiresLike,
-        wires: WiresLike = (),
+        control_wires: WiresLike = "unset",
+        wires: WiresLike = None,
```

Comment:
Why did we change the order back to the original?

Reply:
@astralcai I noticed that there are still some occurrences where people call e.g. `qml.ControlledQubitUnitary(qml.QubitUnitary(np.eye(2), wires=0), [1, 2, 3])` without specifying `control_wires=`. To minimize the impact of this deprecation, it's much better to keep the orignal order so that the legacy usage still gets full support but with PLDW

Reply:
This is even hapenning in our test suites